packages/apps/src/Menu/index.tsx

```diff
@@ -1,13 +1,13 @@
 // Copyright 2017-2020 @polkadot/apps authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
+import { TFunction } from 'i18next';
```

Comment:
Can we configure (and use) a `import type { TFunction } ...` or is it not that aware of syntax? (I guess in this case it works since i18next is generally only touched in the `translate.ts` files, but technically not quite correct.)

Reply:
If I understand correctly, you would like to apply the `import type` syntax to configure linter, so that it separates that type-only imports into another group. If so, then I'm afraid it's not possible yet to configure it that way
![image](https://user-images.githubusercontent.com/30404388/99444892-1949f600-291d-11eb-8344-34462ee5f768.png)